### ISSUE TITLE
Ctrl-X does not work anymore after the update

### ISSUE THREAD

**Issue by aakhmetz** [jupyterlab/jupyterlab#14557]
After the update, Ctrl-X was removed - I used it similarly to Ctrl-D (delete line) when my cursor is blinking on a given line. But now it does not work anymore except if I select the whole line.

I preferred Ctrl-X instead of Ctrl-D because sometimes I would also Cut the lines and put them in other places.

**Comment by aakhmetz** [comment]
Is it possible to get it back? It's extremely inconvenient

**Comment by krassowski** [comment]
It works well for me. Can you provide full reproduction instructions and following the bug report template? Do you have line-wise copy/cut enabled or disabled in CodeMirror settings?

**Comment by aakhmetz** [comment]
It works now after the last update (4.1.6).

It's so cool! Many thanks!
